src/runtime/auto-prompt-manager.ts

```diff
@@ -29,104 +45,82 @@ const TYPE_SUBSCRIPTION = 'TYPE_SUBSCRIPTION';
-   */
-  constructor(deps, configuredRuntime) {
-    /** @private @const {!./deps.Deps} */
-    this.deps_ = deps;
-
-    /** @private @const {!../model/doc.Doc} */
-    this.doc_ = deps.doc();
+  private wasAutoPromptDisplayed_ = false;
+  private hasStoredImpression_ = false;
+  private lastAudienceActionFlow_: AudienceActionFlow | null = null;
+  private interventionDisplayed_: Intervention | null = null;
+
+  private readonly doc_: Doc;
+  private readonly pageConfig_: PageConfig;
+  private readonly entitlementsManager_: EntitlementsManager;
+  private readonly clientConfigManager_: ClientConfigManager;
+  private readonly storage_: Storage;
+  private readonly miniPromptAPI_: MiniPromptApi;
+  private readonly eventManager_: ClientEventManager;
+
+  constructor(
+    private readonly deps_: Deps,
+    private readonly configuredRuntime_: ConfiguredRuntime
+  ) {
+    this.doc_ = deps_.doc();
 
-    /** @private @const {!../model/page-config.PageConfig} */
-    this.pageConfig_ = deps.pageConfig();
+    this.pageConfig_ = deps_.pageConfig();
 
-    /** @private @const {!./entitlements-manager.EntitlementsManager} */
-    this.entitlementsManager_ = deps.entitlementsManager();
+    this.entitlementsManager_ = deps_.entitlementsManager();
 
-    /** @private @const {?./client-config-manager.ClientConfigManager} */
-    this.clientConfigManager_ = deps.clientConfigManager();
+    this.clientConfigManager_ = deps_.clientConfigManager();
     assert(
       this.clientConfigManager_,
       'AutoPromptManager requires an instance of ClientConfigManager.'
     );
 
-    /** @private @const {!./storage.Storage} */
-    this.storage_ = deps.storage();
+    this.storage_ = deps_.storage();
 
-    this.deps_
+    deps_
       .eventManager()
       .registerEventListener(this.handleClientEvent_.bind(this));
 
-    /** @private @const {!MiniPromptApi} */
-    this.miniPromptAPI_ = this.getMiniPromptApi(deps);
+    this.miniPromptAPI_ = this.getMiniPromptApi(deps_);
     this.miniPromptAPI_.init();
 
-    /** @private {boolean} */
-    this.wasAutoPromptDisplayed_ = false;
-
-    /** @private {boolean} */
-    this.hasStoredImpression = false;
-
-    /** @private {?AudienceActionFlow} */
-    this.lastAudienceActionFlow_ = null;
-
-    /** @private {?Intervention} */
-    this.interventionDisplayed_ = null;
-
-    /** @private @const {!./client-event-manager.ClientEventManager} */
-    this.eventManager_ = deps.eventManager();
-
-    /** @private @const {!./runtime.ConfiguredRuntime} */
-    this.configuredRuntime = configuredRuntime;
+    this.eventManager_ = deps_.eventManager();
   }
 
   /**
    * Returns an instance of MiniPromptApi. Can be overwridden by subclasses,
    * such as in order to instantiate a different implementation of
    * MiniPromptApi.
-   * @param {!./deps.Deps} deps
-   * @return {!MiniPromptApi}
-   * @protected
    */
-  getMiniPromptApi(deps) {
+  getMiniPromptApi(deps: Deps): MiniPromptApi {
```

Comment:
Should this method still be marked `protected`?

Reply:
It can even be `private`